.github/.env

```
@@ -7,5 +7,5 @@ ANCHOR_VERSION=0.29.0
 COMMIT_USER_NAME="github-actions"
 COMMIT_USER_EMAIL="github-actions@github.com"
 DEPLOY_SOLANA_VERSION=1.18.19
-RUST_CLIENT_RUST_VERSION=1.82.0
-RUST_CLIENT_SOLANA_VERSION=2.1.21
+RUST_CLIENT_RUST_VERSION=1.89.0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RUST_CLIENT_RUST_VERSION key should go before the RUST_VERSION key

(UnorderedKey)